docs/lang/articles/advanced/layout.md

```diff
@@ -72,54 +51,56 @@ ti.root.dense(ti.i, 3).place(x)
 x = ti.field(ti.f32, shape=3)
 ```
 
-The following declares a 2D field of shape `(3, 4)`:
+* Declare a 2-D field of shape `(3, 4)`:
 
 ```python {1-2}
 x = ti.field(ti.f32)
 ti.root.dense(ti.ij, (3, 4)).place(x)
 # is equivalent to:
 x = ti.field(ti.f32, shape=(3, 4))
 ```
+You can also nest two 1D `dense` statements to describe the same 2D array.
+```python {1-2}
+x = ti.field(ti.f32)
+ti.root.dense(ti.i, 3).dense(ti.j, 4).place(x)
+```
 
-After being comfortable with these equivalent definitions, you can move forward
-and see how to change the data layout.
+In a nutshell, the `ti.root.X` statements progressively binds a shape to the corresponding axis.
```

Comment:
Do we have a concept of axis in field?
```suggestion
In a nutshell, the `ti.root.X` statement progressively binds a shape to the corresponding axis.
```

Reply:
I think ti.i and ti.j / ti.ij are axis, but it has not been mentioned anywhere.

Reply:
@strongoier, is it accurate if we describe like so?

Reply:
How about get rid of axis?

In a nutshell, the `ti.root.X` statement constructs the field by declaring shape at each dimension progressively.

Reply:
Axis is now the official way to talk about `ti.i / ti.j`. But the terms are not completely unified and sometimes people use "dimension / index" to refer to the same thing. We should really have a unique term to avoid confusion.

Reply:
I think these terms are natural. Sometimes the "axis" term reminds people of "scary geometry math", so they just turn to use "dimensions" instead. lol

Reply:
shall we agree on `axis`? Then we can stick to this terminology throughout the docs.

Reply:
Vote for `axis`

Reply:
"dimension" is fine if we use it consistently. Let's go with "dimension".

Reply:
We have explicit APIs for Axis https://github.com/taichi-dev/taichi/blob/cd6d4f82f34f2fa8ec3b14833e0f6732ec987b48/taichi/ir/snode.h#L19-L30

Reply:
@Vissidarte-Herman, let's stick with `axis` for now to be consistent with our code ;/ WDYT?

Reply:
Ugh. Didn't realize it is already in the code. Fine. Let's go with "axis" going forward.